### DISCUSSION TITLE
CUE Community Call 2023-04-12

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2326] [Community Calls]
We are excited to confirm **Wednesday 12 April, 2023 at 1430 UTC** for our next CUE Community Call.

Please reply with any questions/topics you would like to see covered.

### Agenda

- Follow-ups from [Community Call 2023-01-31](https://docs.google.com/presentation/d/1n6tbIv7bHj3WKSnKVrsfVCd49DQFBzhlweXkVzcFo5E/edit)
- Release updates
    - Discussion about release cadence
    - Plan for v0.5.0, v0.6.0 and v0.7.0
- Modules update - including discussion about revised proposal (will be published before the call)
- Unity update
- WebAssembly update
- cuelang.org documentation update
- KubeCon EU and Community updates
- Community Highlight Reel
- Project Ops

In order to give more time to a "Spotlight" CUE user, we are moving this section to its own dedicated session. Our next spotlight will be the [KubeVela](https://kubevela.io/) project, date TBC.

### Meeting details

[https://meet.google.com/gqo-skpa-wpc](https://meet.google.com/gqo-skpa-wpc) 

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar

[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).  

### Recording

https://youtu.be/0Dx5bn2fAaM

### Slides

https://docs.google.com/presentation/d/1h2_a8sO-kHouVKWkxUsQq_-smtGhP3vwM7BSO7oGkXs/edit

**Comment by myitcv** [comment]
I've just tweaked the description above to remove the title `Announcement draft` - this is definitely not a draft! It's the real thing!

**Comment by gdvalle** [comment]
I'd like a blurb on what's planned for performance improvements. What magnitudes of improvement we might hope for, when it's slated, etc. Whatever you can provide would help planning on my end.

**Comment by purpleidea** [comment]
If it's appropriate for the agenda, I'd like to hear about (or if this is already available somewhere, please point me to) what long-term design plans and new features could look like for cue. Thank you!

**Reply by verdverm** [reply]
I've been maintaining a list here: https://cuetorials.com/cueology/futurology/

Happy to add any that I missed

**Reply by myitcv** [reply]
Thanks @verdverm.

See also the answer to FAQ 5 in https://github.com/cue-lang/cue/discussions/2335. 

Milestones in CUE's [issue tracker](https://github.com/cue-lang/cue/issues) _should_ be the source of truth here. We need to update those post the push for v0.5.0 and v0.6.0-alpha.1. Umbrella issues should also capture the background, story etc. 

That's not, of course, to say that other places (e.g. cuetorials) can't also list and share this information. Rather to highlight that we need to do a better job of getting the source information in a better shape!

**Comment by myitcv** [comment]
Thanks to everyone who joined earlier. 

I've just edited the description above to include the video recording and slides links.
